core/src/migrations/000082-models.ts

```diff
@@ -0,0 +1,63 @@
+import Sequelize from "sequelize";
+
+const default_model = "mod_profiles";
```

Comment:
There's a 🐔 🥚 problem here - we want the migration to know how to add a modelId to the tables that need it, but we won't have yet run the initializer to create the first model.  The name is important here.